.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           name: Install dependencies
           command: |
             pip install --upgrade pip twine wheel
-            pip install --editable .[test] --upgrade
+            pip install --editable ".[api, dev]" --upgrade
```

Comment:
With the lastest changes, shouldn't it be `pip install --editable ".[dev]" --upgrade` ?

Reply:
Good catch, forgot that one!